bears/vcs/git/GitRevertInspectBear.py

```diff
@@ -0,0 +1,136 @@
+import re
+
+from difflib import SequenceMatcher
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Result import Result
+
+
+class GitRevertInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    GIT_REVERT_COMMIT_RE = re.compile(
+        r'Revert\s\".+\"\n\nThis\sreverts\scommit\s([0-9a-f]{40})\.')
+
+    def check_modified_file_similarity(self, file_name, reverted_commit_sha):
+        head_commit_branch = run_shell_command(
+            'git rev-parse --abbrev-ref HEAD')[0].strip('\n')
+
+        switch_branch_command = ('git checkout ' +
+                                 head_commit_branch)
+
+        get_revert_commit_diff_command = ('git diff HEAD^ HEAD ' +
+                                          file_name)
+
+        revert_commit_diff = run_shell_command(
+            get_revert_commit_diff_command)[0]
+
+        create_new_branch_command = (
+            'git checkout -b inspectrevertbranch HEAD^')
+        run_shell_command(create_new_branch_command)
+
+        create_expected_revert_commit = ('git revert %s --no-'
+                                         'edit' %
+                                         reverted_commit_sha)
+        try:
+            run_shell_command(create_expected_revert_commit)
```

Comment:
`run_shell_command` won't raise an exception if there's  a content in `stderr`, use `_, err = run_shell_command(...` instead